### ISSUE TITLE
About the Model Neural Network in Part 3 - Model-Based RL.ipynb

### ISSUE THREAD

**Issue by LeonardPatrick** [breeko/Simple-Reinforcement-Learning-with-Tensorflow#2]
Hello Breeko, 

Your code is so clear, very helpful. Just a little question: In the first sentence in Model Neural Network, you said: 
The model neural network will be used to map the current environment and action to a future environment, **action** and reward.
I guess the action maybe should be replaced by done_condition or other words to refer to whether the condition is done or not. 

Cheers,
Jin

**Comment by breeko** [comment]
Thank for pointing out. Issue has been closed.
